packages/legacy/app/App.tsx

```diff
@@ -42,6 +44,10 @@ const App = () => {
     SplashScreen.hide()
   }, [])
 
+  if (!isTablet()) {
```

Comment:
There are some helper functions that use the screen dimensions for this. I think its better if we don't use both. If not in this PR can you create a technical debt ticket to clean this up at a later point?